index.html

```diff
@@ -72,8 +72,198 @@ <h1 class="intro">
         <img src="./Assets/Ellipse4.png" class="roll_ball" alt="roll_ball" />
         <img src="./Assets/Rectangle5.png" class="roll_bar" alt="roll_bar" />
         <img src="./Assets/Ellipse12.png" class="bowl" alt="bowl" />
-        <img src="./Assets/Ellipse13.png" class="hook" alt="hook"/>
+        <img src="./Assets/Ellipse13.png" class="hook" alt="hook" />
       </section>
     </div>
+    <section class="works-container">
+      <div class="work-item">
+        <img
+          class="work-card"
+          src="./Assets/restaurant.JPG"
+          alt="project card display"
+        />
+        <div class="description-container">
+          <h1>Ristorant Confusion</h1>
```

Comment:
- [ ] [**OPTIONAL**] Please let's make sure we use `<h1>` header tag once for any site for SEO. Only for the main header.